src/main/scala/com/exini/dicom/streams/DicomFlows.scala

```diff
@@ -48,6 +48,22 @@ object DicomFlows {
     a
   }
 
+  /**
+    * Flow for stopping processing early once a tag has been reached. Attributes in sequences are ignored.
+    *
+    * @param tag tag number referring to the root dataset at which the processing stop (exclusive)
+    * @return the associated flow
+    */
+  def stopTagFlow(tag: Int): PartFlow =
```

Comment:
Have using a TagPath instead of Int been considered?

Reply:
Great point. Yes I thought about it but figured that this flow is used when you care about performance, and when profiling it seems that creating a tag path for every dicom part puts some stress on the GC. Do you see any obvious use cases where you would like to stop in the middle of a sequence?

Reply:
No, not really.